src/utils/Contracts.js

```diff
@@ -15,7 +15,16 @@ const DEFAULT_COVERAGE_TX_PARAMS = {
 export default {
 
   getLocalPath(contractName) {
-    return `${process.cwd()}/build/contracts/${contractName}.json`
+    let buildDir = `${process.cwd()}/build/contracts`
+    try {
+      const TruffleConfig = require('truffle-config')
+      const config = TruffleConfig.detect({ logger: console })
+      if (config.contracts_build_directory) {
+        buildDir = config.contracts_build_directory
+      }
+    } catch (error) {
+    }
+    return `${buildDir}/${contractName}.json`
```

Comment:
would you mind extracting the Truffle part to a private method? I think it could look cleaner if we do sth like:
```js 
getLocalPath(contractName) {
  const defaultBuildDir = `${process.cwd()}/build/contracts`
  const buildDir = this._getTruffleBuildDir() || defaultBuildDir
  return `${buildDir}/${contractName}.json`
},

_getTruffleBuildDir() {
  try {
    const TruffleConfig = require('truffle-config')
    const config = TruffleConfig.detect({ logger: console })
    return config.contracts_build_directory
  } catch (error) {
    return undefined
  }
}
```